cmd/kubeadm/.import-restrictions

```diff
@@ -0,0 +1,179 @@
+			"AllowedPrefixes": [
+				""
+			]
+		},
+		{
+			"SelectorRegexp": "k8s[.]io/client-go/",
+			"AllowedPrefixes": [
+				""
+			]
+		},
+		{
+			"SelectorRegexp": "k8s[.]io/kube-openapi/",
+			"AllowedPrefixes": [
+				"k8s.io/kube-openapi/pkg/util/proto"
+			]
+		},
+		{
+			"SelectorRegexp": "k8s[.]io/kubernetes/cmd",
+			"AllowedPrefixes": [
+				"k8s.io/kubernetes/cmd/kubeadm"
+			]
+		},
+		{
+			"SelectorRegexp": "k8s[.]io/kubernetes/test",
+			"AllowedPrefixes": [
+				"k8s.io/kubernetes/test/e2e_node/system"
+			]
+		},
+		{
+			"SelectorRegexp": "k8s[.]io/utils",
+			"AllowedPrefixes": [
+				"k8s.io/utils/exec"
+			]
+		},
+		{
+			"SelectorRegexp": "k8s[.]io/kubernetes/pkg",
+			"AllowedPrefixes": [
+				"k8s.io/kubernetes/pkg/api/endpoints",
+				"k8s.io/kubernetes/pkg/api/events",
+				"k8s.io/kubernetes/pkg/api/legacyscheme",
+				"k8s.io/kubernetes/pkg/api/pod",
+				"k8s.io/kubernetes/pkg/api/ref",
+				"k8s.io/kubernetes/pkg/api/resource",
+				"k8s.io/kubernetes/pkg/api/service",
+				"k8s.io/kubernetes/pkg/api/v1/pod",
+				"k8s.io/kubernetes/pkg/apis/admissionregistration",
+				"k8s.io/kubernetes/pkg/apis/admissionregistration/install",
+				"k8s.io/kubernetes/pkg/apis/admissionregistration/v1beta1",
+				"k8s.io/kubernetes/pkg/apis/apps",
+				"k8s.io/kubernetes/pkg/apis/apps/v1",
+				"k8s.io/kubernetes/pkg/apis/apps/v1beta1",
+				"k8s.io/kubernetes/pkg/apis/authentication",
+				"k8s.io/kubernetes/pkg/apis/authentication/install",
+				"k8s.io/kubernetes/pkg/apis/authentication/v1beta1",
+				"k8s.io/kubernetes/pkg/apis/authorization",
+				"k8s.io/kubernetes/pkg/apis/authorization/install",
+				"k8s.io/kubernetes/pkg/apis/autoscaling",
+				"k8s.io/kubernetes/pkg/apis/batch",
+				"k8s.io/kubernetes/pkg/apis/batch/install",
+				"k8s.io/kubernetes/pkg/apis/batch/v1beta1",
+				"k8s.io/kubernetes/pkg/apis/batch/v2alpha1",
+				"k8s.io/kubernetes/pkg/apis/certificates",
+				"k8s.io/kubernetes/pkg/apis/componentconfig",
+				"k8s.io/kubernetes/pkg/apis/core",
+				"k8s.io/kubernetes/pkg/apis/core/helper",
+				"k8s.io/kubernetes/pkg/apis/core/pods",
+				"k8s.io/kubernetes/pkg/apis/core/v1",
+				"k8s.io/kubernetes/pkg/apis/core/v1/helper/qos",
+				"k8s.io/kubernetes/pkg/apis/core/validation",
+				"k8s.io/kubernetes/pkg/apis/events",
+				"k8s.io/kubernetes/pkg/apis/extensions",
+				"k8s.io/kubernetes/pkg/apis/networking",
+				"k8s.io/kubernetes/pkg/apis/networking/v1",
+				"k8s.io/kubernetes/pkg/apis/policy",
+				"k8s.io/kubernetes/pkg/apis/rbac",
+				"k8s.io/kubernetes/pkg/apis/rbac/v1",
+				"k8s.io/kubernetes/pkg/apis/scheduling",
+				"k8s.io/kubernetes/pkg/apis/settings",
+				"k8s.io/kubernetes/pkg/apis/settings/install",
+				"k8s.io/kubernetes/pkg/apis/settings/v1alpha1",
+				"k8s.io/kubernetes/pkg/apis/storage",
+				"k8s.io/kubernetes/pkg/capabilities",
+				"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset",
```

Comment:
I really hope we can get rid of this some day... Could you open an issue to track that we should get rid of the dep of the internal clientset as we are using client-go?

Reply:
yes i can

Reply:
https://github.com/kubernetes/kubeadm/issues/806

Reply:
Thanks!

Reply:
Thanks!